.github/test_dependencies.yaml

```diff
@@ -1,26 +1,40 @@
 e2e:
   kind:
+    # renovate: datasource=docker depName=kindest/node versioning=docker
     - 'v1.28.0'
+    # renovate: datasource=docker depName=kindest/node versioning=docker
     - 'v1.27.3'
+    # renovate: datasource=docker depName=kindest/node versioning=docker
     - 'v1.26.6'
+    # renovate: datasource=docker depName=kindest/node versioning=docker
     - 'v1.25.11'
+    # renovate: datasource=docker depName=kindest/node versioning=docker
     - 'v1.24.15'
+    # renovate: datasource=docker depName=kindest/node versioning=docker
     - 'v1.23.17'
   gke:
     - 'v1.26.5'
+
+  # For Istio, we define combinations of Kind and Istio versions that will be
+  # used directly in the test matrix `include` section.
   istio:
-    # For Istio, we define combinations of Kind and Istio versions that will be
-    # used directly in the test matrix `include` section.
     - kind: 'v1.28.0'
```

Comment:
Seems that we should also update kind version here?

Reply:
We might need to add the code markers to this as well. We can do this as we go